torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -54,20 +54,72 @@ static IValue Table(
 }
 
 std::string getModulePath(Node* node) {
-  std::string modulePath = node->scopeName();
-  size_t end = modulePath.size();
-  // Here we remove the source range information to make the
-  // module debugging information shorter and cleaner.
-  if (modulePath[end - 1] == '>') {
-    end = modulePath.rfind('<');
-    if (end > 0 && modulePath[end - 1] == '<') {
-      --end;
+  auto callStack = node->callstack();
+  return callStack.has_value() ? callStack->get()->getModulePath() : "";
+}
+
+std::string getModuleTypeName(const Module& module, const std::string& prefix) {
```

Comment:
nit: If it's only used in this file, please use anonymous namespace to mark it "local".

Reply:
nvm. It is already in anonymous namespace. My bad...